### PR TITLE
Update database-persistence.md

### PR DESCRIPTION
**Pull request by AlonMagrafta**
Update database-persistence for RoF / Auto Tiering

### CODE REVIEW

File: content/rs/databases/configure/database-persistence.md [hunk 1]
```diff
@@ -104,26 +104,25 @@ crdb-cli crdb update --crdb-guid <CRDB_GUID> --default-db-config \
...
 and network overhead, especially for cloud configurations
 with network-attached persistent storage, such as EBS-backed
 volumes in AWS.
 
 There may be times when performance is critical for your use case and
-you don't want to risk data persistence adding latency. If that is the
-case, you can disable data-persistence on the master shards using the
+you don't want to risk data persistence adding latency.
+
+You can enable or disable data-persistence on the master shards using the
```
Comment (rrelledge) [inline content/rs/databases/configure/database-persistence.md:123]:
```suggestion
You can enable or turn off data persistence on the master shards using the
```

File: content/rs/databases/configure/database-persistence.md [hunk 2]
```diff
@@ -104,26 +104,25 @@ crdb-cli crdb update --crdb-guid <CRDB_GUID> --default-db-config \
...
-If you are enabling data persistence for databases running on Redis
-Enterprise Flash, by default both master and replica shards are
-configured to write to disk. This is unlike a standard Redis Enterprise
-Software database where only the replica shards persist to disk. This
-master and replica dual data persistence with replication is done to
-better protect the database against node failures. Flash-based databases
-are expected to hold larger datasets and repair times for shards can
-be longer under node failures. Having dual-persistence provides better
-protection against failures under these longer repair times.
+Auto Tiering Flash storage is not considered as persistent storage.
```
Comment (rrelledge) [inline content/rs/databases/configure/database-persistence.md:107]:
```suggestion
Auto Tiering flash storage is not considered persistent storage.
```

File: content/rs/databases/configure/database-persistence.md [hunk 3]
```diff
@@ -104,26 +104,25 @@ crdb-cli crdb update --crdb-guid <CRDB_GUID> --default-db-config \
...
+
+Flash-based databases are expected to hold larger datasets and repair
+times for shards can be longer under node failures. To better protect
+the database against node failures, users might consider enabling
+master and replica dual data persistence provides better protection
+against failures under these longer repair times.
```
Comment (rrelledge) [inline content/rs/databases/configure/database-persistence.md:113]:
```suggestion
Flash-based databases are expected to hold larger datasets, and shard repair times can take longer after node failures. To better protect the database against node failures with longer repair times, consider enabling master and replica dual data persistence.
```

**Review by rrelledge — APPROVED** [review]
Looks good to me. I just left some wording suggestions to consider.

**Comment by AlonMagrafta** [comment]
@rrelledge thanks! approved ;)

**Pull request merged**